File: cours10/exo1.py
firstname = "Jean-Baptiste"

lengthFirstname = len(firstname);

print( f"Votre prénom a {lengthFirstname} caractères")


first3LetterFirstname = firstname[0:3]
print(first3LetterFirstname)



jeanInFirstname = firstname.find("Jean")

if jeanInFirstname == -1:
  print("Pas de Jean dans le prénom")

else:
  print(f"Jean à la position   {jeanInFirstname}")

File: cours10/exo1.php
<?php

$firstname = "Jean-Baptiste";

$lengthFirstname = strlen($firstname);

echo "Votre prénom a ".$lengthFirstname." caractères \n";


$first3LetterFirstname = substr($firstname, 0, 3);
echo $first3LetterFirstname."\n";



$jeanInFirstname = strpos($firstname, "Jean");

if($jeanInFirstname === false) {
  echo "Pas de Jean dans le prénom";
}
else {
  echo "Jean à la position ".$jeanInFirstname;
}








 ?>
